tests/secret-providers/aws_secretmanager/aws_secretmanager_test.go

```diff
@@ -0,0 +1,452 @@
+//go:build e2e
+// +build e2e
+
+package aws_secret_manager_test
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/secretsmanager"
```

Comment:
Same as above, we have to use the sdk-v2

Reply:
Done